types/bitcore-lib/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es6"],
+    "target": "es6",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": false,
```

Comment:
For new packages, this should be enabled

Reply:
Thanks @plantain-00 ! I've pushed the change now